images/prow-tests/Dockerfile

```diff
@@ -27,6 +27,7 @@ RUN docker-credential-gcr configure-docker
 # Extra tools through apt-get
 RUN apt-get install -y uuid-runtime  # for uuidgen
 RUN apt-get install -y npm  # for markdownk-link-check
+RUN apt-get install -y build-essential libssl-dev git # for wrk(perf benchmarking tool) 
```

Comment:
git is already available in the image, you shouldn't need anything new to `git clone` stuff.

Reply:
removed